course/admin.py

```diff
@@ -591,8 +591,7 @@ def get_participant(self, obj):
     search_fields = (
             "flow_id",
             "participation__user__username",
-            "participation__user__first_name",
-            "participation__user__last_name",
+            "participation__user__full_name",
```

Comment:
Disagreed here. This'll either not work or make the search very expensive--because the database can't carry it out on its own.